website/docs/testing/markdown/doc-components.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+import TOKENS_RAW from '@hashicorp/design-system-tokens/dist/docs/products/tokens.json';
+
+const ELEVATIONS = ['inset', 'low', 'mid', 'high', 'higher', 'overlay'];
+
+export default class Index extends Component {
+  get colors() {
+    const colors = {
+      semantic: {},
+    };
+
+    TOKENS_RAW.forEach((token) => {
+      if (token.attributes.category === 'color' && !token.deprecated) {
+        if (token.group) {
+          if (token.group === 'semantic') {
+            const context = token.path[1];
+            if (!colors['semantic'][context]) {
+              colors['semantic'][context] = [];
+            }
+            const tokenObj = {
+              colorName: `${token.path[1]}-${token.path[2]}`,
+              cssVariable: `--${token.name}`,
+              value: token.value,
+            };
+            if (['foreground', 'page', 'surface', 'border'].includes(context)) {
+              const name = token.path[2];
+              tokenObj.cssHelper = `hds-${context}-${name}`;
+            }
+            colors['semantic'][context].push(tokenObj);
+          }
+        } else {
+          console.log(
+            `Notice: the color ${token.name} is missing the 'group' attribute in the JSON definition.`
+          );
+        }
+      }
+    });
+
+    return colors;
+  }
+
+  get cssHelpers() {
+    const cssHelpers = { elevations: [] };
+    ELEVATIONS.forEach((elevation) => {
+      cssHelpers.elevations.push(`.hds-elevation-${elevation}`);
+    });
+    return cssHelpers;
+  }
```

Comment:
Let's keep it simple and minimal
```suggestion
  get varsListDemo() {
    return ['lorem-ipsum'];
  }
```

Reply:
Absolutely,  my efforts in this test file were not intended to be over engineered. They were two-fold... Ensure that inline rendered components were wrapped in p ( acceptance criteria changed late in the game ) and to ensure that all block custom elements were not wrapped in p tags. Secondly was to master ember quickly and get acclimated to the way the team codes so I can fall in line, follow the same patterns, and create consistent experiences. That being said, I see that it is a huge priority to keep these test files perfect and minimal, I will ensure that my future PRs comply to this standard.

Reply:
> I see that it is a huge priority to keep these test files perfect and minimal

It's not about the test files, it's about the reviewer(s): reviewing 100 lines of code instead of 500 (generic numbers) makes a substantial difference.

As general approach, in the context of our design system, yes we tend to err on the "less is more" side, if something is not needed. That's why we differentiate between spikes in code (free for all) and actual code that gets merged in `main`.

Reply:
And now I know "The Why" ... makes total sense. Thank you for the additional context, it is very helpful.